tests/fixtures.py

```diff
@@ -57,6 +57,13 @@ class UrlFormTest(forms.Form):
         filters=[forms.filters.strip()],
     )
 
+class EmojiFormTest(forms.Form):
+    emoji = forms.StringField(
+        __("Emoji"),
+        validators=[
+        forms.validators.IsEmoji()
```

Comment:
this doesn't seem black compliant. If you have black setup, format this file once.